### PR TITLE
[15.0][IMP] auditlog: make the line views using a non auto model

### PR DESCRIPTION
**Pull request by kluna1998**
https://github.com/OCA/server-tools/pull/2509 ported

### CODE REVIEW

**Review by etobella — APPROVED** [review]

File: auditlog/tests/test_auditlog.py [hunk 1]
```diff
@@ -332,6 +332,7 @@ def test_01_field_and_model_removal(self):
 
     def test_02_migration(self):
         """Test the migration of the data model related to this feature"""
+        self.env.cr.execute("""DROP VIEW auditlog_log_line_view""")
```
Comment (pedrobaeza) [inline auditlog/tests/test_auditlog.py]: I think all this test should go out, as even the migration script shouldn't be here in this branch.

File: auditlog/security/ir.model.access.csv [hunk 2]
```diff
@@ -11,3 +11,5 @@ access_auditlog_log_line_manager,auditlog_log_line_manager,model_auditlog_log_li
 access_auditlog_http_session_manager,auditlog_http_session_manager,model_auditlog_http_session,auditlog.group_auditlog_manager,1,1,1,1
 access_auditlog_http_request_manager,auditlog_http_request_manager,model_auditlog_http_request,auditlog.group_auditlog_manager,1,1,1,1
 access_auditlog_autovacuum,access_auditlog_autovacuum,model_auditlog_autovacuum,auditlog.group_auditlog_user,1,1,1,1
+
```
Comment (pedrobaeza) [inline auditlog/security/ir.model.access.csv]: No empty lines, please

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge major

**Comment by pedrobaeza** [comment]
Please do it also on v16

**Pull request merged**